config.go

```diff
@@ -527,6 +532,8 @@ type Config struct {
 	// NumRestrictedSlots is the number of restricted slots we'll allocate
 	// in the server.
 	NumRestrictedSlots uint64 `long:"num-restricted-slots" description:"The number of restricted slots we'll allocate in the server."`
+
+	DisconnectOnPongFailure bool `long:"disconnect-on-pong-failure" description:"Whether to disconnect a peer if a pong is not received in time or is mismatched. Defaults to false."`
```

Comment:
Not sure if we need the "default to false" sentence here, since all our boolean flags behave that way?

Reply:
Removed, was sort second guessing myself re some of the quirks w/ the parser and bool args.